examples/lsp_spice.py

```diff
@@ -66,20 +68,54 @@
 shape_model = TrimeshShapeModel(V, F, N)
 
 # Load compressed form factor matrix from disk
-
 FF_path = 'lsp_compressed_form_factors.bin'
 FF = cff.CompressedFormFactorMatrix.from_file(FF_path)
+# print("Getting FF block") # what's this?
+# FF_gt = get_form_factor_block(shape_model)
 
-FF_gt = get_form_factor_block(shape_model)
-
-# Compute steady state temperature
 E_arr = []
 for i, sun_dir in enumerate(sun_dirs[:]):
     E_arr.append(shape_model.get_direct_irradiance(F0, sun_dir))
 
 E = np.vstack(E_arr).T
-T_arr = compute_steady_state_temp(FF, E, rho, emiss)
-T = np.vstack(T_arr).T
+
+if not subsurf_heat:
+    # Compute steady state temperature
+    T_arr = compute_steady_state_temp(FF, E, rho, emiss)
+    T = np.vstack(T_arr).T
+else:
```

Comment:
maybe it would be useful to have the content of this `else` in a separate function to be also used somewhere else (how to best structure it)?